build/hellobuild.md

```diff
@@ -0,0 +1,135 @@
+---
+
+## Hello Build!
+
+It all starts with a Dockerfile.
+
+Dockerfiles are text files containing instructions. Dockerfiles adhere to a specific format and contain a **set of instructions** for which you can find a full reference in the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/).
+<br/>Docker builds images by reading the instructions from a Dockerfile.
+
+Docker images consist of **read-only layers**, each resulting from an instruction in the Dockerfile. Layers are stacked sequentially and each one is a delta representing the changes applied to the previous layer.
+
+## Dockerfile basics
+
+The Dockerfile is a text file containing all necessary instructions needed to assemble and deploy a container image with Docker.
+
+Here are the most common types of instructions:
+ 
+* [**FROM \<image\>**](https://docs.docker.com/engine/reference/builder/#from) - defines a base for your image.
+* [**RUN \<command\>**](https://docs.docker.com/engine/reference/builder/#run) - executes any commands in a new layer on top of the current image and commits the result. <br/>RUN also has a shell form for running commands.
+* [**WORKDIR \<directory\>**](https://docs.docker.com/engine/reference/builder/#workdir) - sets the working directory for any RUN, CMD, ENTRYPOINT, COPY and ADD instructions that follow it in the Dockerfile.
+* [**COPY \<src\> \<dest\>**](https://docs.docker.com/engine/reference/builder/#copy) - copies new files or directories from \<src\> and adds them to the filesystem of the container at the path \<dest\>.
+* [**CMD \<command\>**](https://docs.docker.com/engine/reference/builder/#cmd) - lets you define the default program that is run once you start the container based on this image. Each Dockerfile only has one CMD, and only the last CMD instance is respected when multiple exist.
+<br/>Dockerfiles are crucial inputs for image builds and can facilitate automated, multi-layer image builds based on your unique configurations. Dockerfiles can start simple and grow with your needs and support images that require complex instructions.
+  
+For all the possible instructions, see the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/).
+
+## Example
+Here’s a simple Dockerfile example to get you started with building images. We’ll take a simple "Hello World" Python Flask application, and bundle it into a docker image that we can test locally or deploy anywhere!
+
+**Sample A**
+<br/>Let’s say we have the following in a `hello.py` file in our local directory:
+
+{% highlight python lineos %}
+from flask import Flask
+app = Flask(__name__)
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+{% endhighlight %}
+
+Don’t worry about understanding the full example if you’re not familiar with Python - it’s just a simple web server that will contain a single page that says “Hello World”. 
+
+> **Note**: If you test the example, make sure to copy over the indentation as well!
```

Comment:
@dockertopia Could you use the standard note format?

Reply:
Sure, I'll reach out to you to clarify standard.

Reply:
Fixed the typo.